core/src/main/scala/org/apache/spark/util/Clock.scala

```diff
@@ -22,8 +22,35 @@ package org.apache.spark.util
  */
 private[spark] trait Clock {
```

Comment:
Would be good to have "Millis" in the name to make the time scale clear. For some clocks the time scale matters.